net/httping/Portfile

```diff
@@ -27,15 +28,21 @@ long_description    Give it an url, and it will show you how long it takes to \
 
 github.tarball_from archive
 
-depends_lib         port:fftw-3 \
-                    port:gettext \
-                    port:ncurses \
-                    path:lib/libssl.dylib:openssl
+depends_build-append \
+                    port:gettext
+
+depends_lib-append  port:fftw-3 \
+                    port:gettext-runtime \
```

Comment:
you are changing the `depends_lib` dependencies so that requires to increase the `revision`